wrappers/javascript/packages/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -102,6 +102,16 @@ export class NodeJSAnoncreds implements Anoncreds {
     return handleReturnPointer<string>(ret)
   }
 
+  public w3cCredentialGetAttribute(options: { objectHandle: ObjectHandle; name: string }) {
```

Comment:
Is there a reason why there is a separate method for w3c? Getting an attribute is the same for w3c and non w3c credentials right?

Reply:
I do not understand your point. `w3cCredentialGetAttribute` method calls `anoncreds_w3c_credential_get_attribute` external Rust function which obtain W3C credential object from handle and returns value for requesting attribute.

Reply:
Yeah I was thinking more of that whether the object is an AnonCreds representation or a W3C representation doesn't really change the input or output of this method, so from an user POV it could be the same method right?